From 0d7df68a741f8e91c3dc86d9fc3020c3711bc0f2 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?R=C3=A9mi=20Cardona?= <remi.cardona@lri.fr>
Date: Thu, 20 Nov 2008 17:58:15 +0100
Subject: [PATCH 03/41] Turn AllowEmptyInput off by default if HAL support is disabled

This allows users without HAL support to have working "mouse" and "kbd"
without tweaking their xorg.conf
---
 hw/xfree86/common/xf86Globals.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/hw/xfree86/common/xf86Globals.c b/hw/xfree86/common/xf86Globals.c
index a5edd82..6fc5a9a 100644
--- a/hw/xfree86/common/xf86Globals.c
+++ b/hw/xfree86/common/xf86Globals.c
@@ -134,7 +134,11 @@ xf86InfoRec xf86Info = {
     .kbdCustomKeycodes          = FALSE,
     .disableRandR               = FALSE,
     .randRFrom                  = X_DEFAULT,
+#ifdef CONFIG_HAL
     .allowEmptyInput            = TRUE,
+#else
+    .allowEmptyInput            = FALSE,
+#endif
     .autoAddDevices             = TRUE,
     .autoEnableDevices          = TRUE
 };
-- 
1.6.0.4

